cherrypicks_test.go

```diff
@@ -394,11 +394,60 @@ func TestParseCherryTargetBranches(t *testing.T) {
 		* 1.2.x\r`,
 			expected: "1.2.x",
 		},
+		"multiple branches": {
+			body: `cherry-pick to:
+		* master
+		* hosted
+		* example-branch`,
+			expected: "master hosted example-branch",
+		},
+	}
+
+	for name, test := range tests {
+		t.Log(name)
+		res, _ := parseCherryTargetBranches(test.body)
+		assert.Equal(t, test.expected, strings.Join(res, " "))
```

Comment:
Suggestion:

Change the type to the type returned by the function instead of joining here. It really is what we want to test.

Reply:
I guess so, I thought perhaps having `expected: []string {"expected-output"}` for each entry would be a pain to the eyes.